README.md

```diff
@@ -1 +1,171 @@
-# d2-docker
\ No newline at end of file
+## Requirements
+
+-   Python >= 3.5 (with python3-distutils)
+-   Docker >= 18
+-   Docker compose >= 1.17
+
+## Usage
+
+### Start a DHIS2 instance
+
+Start a new container from a _dhis2-data_ base image:
+
+```
+$ bin/d2-docker start eyeseetea/dhis2-data:2.30-sierra
```

Comment:
What If I want to start a docker without preloaded data? This should be a common use case at least for the readme

Reply:
Good point! I will add this feature.